src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -141,10 +148,10 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
 
         // keep height undef for AD; causes indigo-visualizations to pick default 100%
         const resultingHeight = this.environment === DASHBOARDS_ENVIRONMENT ? height : undefined;
-
         const resultSpec = this.getResultSpec(options, visualizationProperties, mdObject);
-
-        const fullConfig = this.buildVisualizationConfig(mdObject, config, null);
+        const supportedControls: IVisualizationProperties = get(visualizationProperties, "controls", {});
+        const configSupportedControls = isEmpty(supportedControls) ? null : supportedControls;
```

Comment:
this could be shorter 
`const configSupportedControls = !isEmpty(supportedControls) && supportedControls;`

Reply:
updated